middleware/basic_auth_test.go

```diff
@@ -25,20 +26,36 @@ func TestBasicAuth(t *testing.T) {
 	h := BasicAuth(f)(func(c echo.Context) error {
 		return c.String(http.StatusOK, "test")
 	})
+	s := func(c echo.Context) bool {
```

Comment:
Tests do not need to be that complex. You could add skipper that always skips (`return true`) and verify that it skips.

Reply:
revised boss. @aldas kindly review again